compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1848,8 +1850,104 @@ setRootConfig(mlir::FunctionOpInterface entryPointFn,
                              vecPreProcStrategy);
 }
 
+// Utility function to return the result inner tile sizes associated with the
+// mmt4d op. I.e. returns the M1 and N1 dimensions of the result shape
+// BxM0xN0xM1xN1. In case of scalable vectors, walks the IR and returns
```

Comment:
Note, there are no vectors at this level. Also, could you document what happens for static sizes?
```suggestion
// BxM0xN0xM1xN1. In case of scalable dimensions, walks the IR and returns
```